module-system/sov-modules-macros/tests/all_tests.rs

```diff
@@ -3,12 +3,13 @@ fn module_info_tests() {
     let t = trybuild::TestCases::new();
     t.pass("tests/module_info/parse.rs");
     t.pass("tests/module_info/mod_and_state.rs");
+    t.compile_fail("tests/module_info/derive_on_enum_not_supported.rs");
     t.compile_fail("tests/module_info/field_missing_attribute.rs");
+    t.compile_fail("tests/module_info/missing_address.rs");
+    t.compile_fail("tests/module_info/no_generics.rs");
```

Comment:
Does adding a test for the `ModuleCallJsonSchema` macro make sense?  Maybe we could tests the `Failed to serialize JSON schema; this is a bug in the module` message?